### ISSUE TITLE
triggers UI

### ISSUE THREAD

**Issue by jedgalbraith** [nanobox-io/-nanobox-io-nanobox-dash-ui-valkrie#7]
We need a UI that will allow users to create triggers on hosts and components. I'm guessing this will be part of the valkrie component. Trigger fields include:

`name(string)` - User defined name for their benefit.

`metric(string)` - The server resource the trigger should be based on. Options include: `disk_percent, ram_percent, cpu_percent`. Odin can provide these options along with labels during component instantiation.

`action(string)` - What should happen when triggered. Currently the only option is `email`. Odin can also provide these options.

`threshold(float)` - The metric value, representing a percentage, that when exceeded will trigger the configured action. Note: Odin will translate the float to a string before sending to Pulse(e.g. `0.8` will be translated to `'0.8'`).

`duration(int)` - The length of time the threshold should be exceeded before being triggered. Value should be in seconds. Note: Odin will translate this value to the format expected by Pulse/Kapacitor(e.g. `30` will be translated to `'30s'`).

`triggerable_type` - Identifies the type of subject. Either `host` or `component`.

`triggerable_id` - A reference to the subject of the trigger. Either a host or component uuid.

**Comment by Tolmark12** [comment]
Yes let's talk, I have the design for this. I also started the actual component a few months back. It's a stand-alone component not part of valkrie.

![image](https://cloud.githubusercontent.com/assets/4827/25205046/9f51e544-251d-11e7-94cf-e309dc5ea830.png)
